wdl2cwl/tests/cwl_files/seqtk.cwl

```diff
@@ -5,6 +5,14 @@ inputs:
     type: File
   - id: fractionOrNumber
     type: float
+  - id: preCommand
+    type:
+      - string
+      - 'null'
```

Comment:
If we don't provide a "preCommand" what goes in the bash script for `$(inputs.preCommand)` ? Is it an empty string or the word "null"? 

If it is the later, then we will have a problem and we should set the default value for optional strings to `""`

Reply:
@dinithiW can you respond to my question above?

Reply:
I didn't quite get back this PR again. I tried running with a Fasta file to which I got a command failed error

Reply:
Can you share the error? More importantly, can you run with `cwltool --cachedir some_path wdl2cwl/tests/cwl_files/seqtk.cwl ` and share the rendered `example.sh`?

Reply:
resultant example.sh
```
set -e -o pipefail
        mkdir -p "$(dirname subsampledReads.fq.gz)"
        null
        seqtk sample 
	    -s null 
        /tmp/77vey6du/stg5cc2fcb9-f851-4728-ae19-afa32d16efaf/insert_size_metrics_test.sam 
        1.0 
        >  subsampledReads.fq.gz
```

Reply:
Okay, just as I feared.

To quote myself:

> If it is the later, then we will have a problem and we should set the default value for optional strings to `""`

Reply:
Ok. I will find a workaround for that.